mobx_codegen/lib/src/store_class_visitor.dart

```diff
@@ -1,7 +1,6 @@
 // https://github.com/dart-lang/sdk/blob/main/pkg/analyzer/doc/element_model_migration_guide.md
-// ignore_for_file: deprecated_member_use
 
-import 'package:analyzer/dart/element/element2.dart';
+import 'package:analyzer/dart/element/element.dart';
 import 'package:analyzer/dart/element/visitor2.dart';
```

Comment:
`visitor2.dart` is not deprecated as of analyzer 8.1.1 and `visitor.dart` still points to the old API, so no change here yet.